### PR TITLE
[onnx_utils] update torchvision version 

### PR DESCRIPTION
**Pull request by daniels290813**

### CODE REVIEW

**Review by xsqian — APPROVED** [review]

File: onnx_utils/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 tqdm~=4.62.3
 tensorflow~=2.7.0
-torch~=1.10.0
-torchvision~=0.11.1
+torch==1.10.0
+torchvision==0.11.1
```
Comment (guy1992l) [inline onnx_utils/requirements.txt:4]: why the freezing of the version? if it's not working with 1.10.x we should check it and see why, not freeze version and ignore it.

File: onnx_utils/function.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ spec:
     base_image: mlrun/ml-models
     commands:
     - python -m pip install onnx~=1.10.1 onnxruntime~=1.8.1 onnxoptimizer~=0.2.0 onnxmltools~=1.9.0
-      tf2onnx~=1.9.0
+      tf2onnx~=1.9.0 torchvision==0.11.1
```
Comment (guy1992l) [inline onnx_utils/function.yaml:20]: why is `torchvision` required?

**Review by guy1992l — CHANGES_REQUESTED** [review]

**Comment by daniels290813** [comment]
sorry should be in the description 
running mask-detection demo fails with
```
from torchvision.transforms import InterpolationMode
ImportError: cannot import name 'InterpolationMode' from 'torchvision.transforms' (/opt/conda/lib/python3.7/site-packages/torchvision/transforms/__init__.py)
```

mlrun/ml-models image uses the following versions
```
# pytorch caused cuda download, this guarantees cpu version:
RUN python -m pip install torch==1.7.0+cpu torchvision==0.8.1+cpu \
```
apparently this functionality is added in latter version, check this out
https://blog.csdn.net/prague6695/article/details/120457004

**Comment by aviaIguazio** [comment]
replace the InterpolationMode import by passing an integer or string that represent the enum value. In older versions it worked with PIL modes and torch decided they want their own. He can still give either "nearest" string or 0 integer and avoid importing the InterpolationMode.

**Pull request closed without merging**